docs/fleet/_index.md

```diff
@@ -18,103 +18,62 @@ aliases:
 menuindent: true
 ---
 
-The [Viam app](https://app.viam.com) provides fleet management allowing you to work on any number of machines alone or in collaboration with others.
-You can manage and control your fleet of machines from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
+Viam fleet management allows you to organize, manage, and control any number of machines alone or in collaboration with others.
+You can manage and control your fleet of {{< glossary_tooltip term_id="machine" text="smart machines" >}} from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
 
 ## Work with groups of machines
 
-With Viam, you can organize {{< glossary_tooltip term_id="machine" text="smart machines" >}} into {{< glossary_tooltip term_id="location" text="locations" >}} and {{< glossary_tooltip term_id="organization" text="organizations" >}}.
+To organize your fleet you use
 
-For example, you may have separate organizations for your machines at home and at work:
+- {{< glossary_tooltip term_id="organization" text="organizations" >}}: the highest level grouping, generally used for different companies.
+- {{< glossary_tooltip term_id="location" text="locations" >}}: virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
```

Comment:
I wonder if we should have a "machines" bullet and explain that is the most important thing - ie the thing you are organizing around

Reply:
Added